impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Funnel.js

```diff
@@ -0,0 +1,179 @@
+/*!
+ * Copyright 2010 - 2023 Hitachi Vantara. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "pentaho/module!_",
+  "pentaho/visual/impl/View",
+  "common-ui/echarts",
+  "../../../ccc/visual/_util"
+], function(module, BaseView, echarts, util) {
+
+  "use strict";
+
+  return BaseView.extend(module.id, {
+
+    _configureLabel: function (option) {
+      var font = util.defaultFont(null, 12);
+
+      var label = {
+        show: true,
+        position: this.model.labelsOption,
+        formatter: '{b}: {d}%',
+        backgroundColor: 'transparent',
+        fontSize: font.substring(0, font.indexOf(' ')),
+        fontFamily: font.substring(font.indexOf(' ') + 1)
+      };
+
+      option.series.forEach(function (row) {
+        row.label = label;
+      })
```

Comment:
This file is not following the formatting conventions of the surrounding files. For example on the use of quotes, or the spacing around parenthesis in function expressions. This project has a `.eslintrc.json` file at the root of the project which specifies the followed convention in this area of the code. IntellIiJ can be configured to use it.